ios/FluentUI.Demo/FluentUI.Demo/Demos/AvatarGroupDemoController_SwiftUI.swift

```diff
@@ -0,0 +1,176 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import FluentUI
+import UIKit
+import SwiftUI
+
+class AvatarGroupDemoControllerSwiftUI: UIHostingController<AvatarGroupDemoView> {
+    override init?(coder aDecoder: NSCoder, rootView: AvatarGroupDemoView) {
+        preconditionFailure("init(coder:) has not been implemented")
+    }
+
+    @objc required dynamic init?(coder aDecoder: NSCoder) {
+        preconditionFailure("init(coder:) has not been implemented")
+    }
+
+    init() {
+        super.init(rootView: AvatarGroupDemoView())
+        self.title = "Avatar Group Vnext (SwiftUI)"
+    }
+}
+
+struct AvatarGroupDemoView: View {
+    // AvatarGroup properties
+    @State var avatarCount: Int = 5
+    @State var style: MSFAvatarGroupStyle = .stack
+    @State var maxDisplayedAvatars: Int = 5
+    @State var overflowCount: Int = 0
+
+    // Avatar properties
+    @State var useAlternateBackground: Bool = false
+    @State var isRingVisible: Bool = true
+    @State var isTransparent: Bool = true
+    @State var hasRingInnerGap: Bool = true
+    @State var showImageBasedRingColor: Bool = false
+    @State var size: MSFAvatarSize = .xxlarge
+
+    // TODO: Switch to Fluent button once it's available in SwiftUI
+    struct AvatarCountButtonStyle: ButtonStyle {
+        func makeBody(configuration: Configuration) -> some View {
+            configuration.label
+                .frame(width: 32, height: 32)
+                .foregroundColor(Color(Colors.communicationBlue))
+                .background(
+                    RoundedRectangle(cornerRadius: 8.0)
+                        .strokeBorder(Color(Colors.communicationBlue))
+                )
+        }
+    }
+
+    @ViewBuilder
+    var groupSettings: some View {
+        Group {
+            DemoHeading(title: "Group Settings")
+
+            Picker(selection: $style, label: EmptyView(), content: {
+                Text(".pile").tag(MSFAvatarGroupStyle.pile)
+                Text(".stack").tag(MSFAvatarGroupStyle.stack)
+            })
+            .pickerStyle(SegmentedPickerStyle())
+            .labelsHidden()
+            .frame(maxWidth: .infinity, alignment: .leading)
+
+            HStack {
+                Text("Avatar Count")
+
+                Button("+") {
```

Comment:
can we not use text + it looks bad in larger text? either use the image or use the actual stepper? also nit pick - on left and + on right?

Reply:
Something that was also mentioned by @rdeassis offline was maybe using the AG protocol methods (create/remove avatars) to determine the avatar count?

Reply:
@harrieshin agreed! I'd pulled those from the previous demo but I much prefer your suggestions.

@sophialee0416 good idea! those are not being tested right now, and this would be a great chance to do so.

Reply:
@sophialee0416 Actually following up on your feedback: this PR removes those APIs from `AvatarGroup` directly and moves them to the `MSFAvatarGroup` wrapper, so there's no concept of calling `create`/`get`/`remove` from the SwiftUI object's state. 

Instead, users will simply update `count` to reflect the new count of avatars, and the generator will re-run to fetch the new values. 

Meanwhile, updating the properties of a single Avatar are done by simply updating the properties that were used to generate that Avatar, and SwiftUI will manage the refresh automatically. This is what we're doing with the various `avatarSettings` toggles in this file.

Let me know if you want more info and I'd be happy to sit down and go over it all in detail!